packages/migrate/src/commands/DbSeed.ts

```diff
@@ -39,6 +45,11 @@ ${bold('Options')}
     )
 
     if (isError(args)) {
+      if (args instanceof ArgError && args.code === 'ARG_UNKNOWN_OPTION') {
+        throw new Error(`${args.message}
+Did you mean to pass these as arguments to your seed script? If so, add a -- separator before them:
+${dim('$')} prisma db seed -- --arg1 value1 --arg2 value2`)
```

Comment:
In theory this could even use the real arg that created the error :D

Reply:
We could, but then we would also need to parse the key/values to read the value, which we would "leak" in this error message. It's easy enough for the person to add a `--` where needed here.